doc/api/vm.markdown

```diff
@@ -301,6 +301,34 @@ e.g. `(0,eval)('code')`. However, it also has the following additional options:
 - `timeout`: a number of milliseconds to execute `code` before terminating
   execution. If execution is terminated, an [`Error`][] will be thrown.
 
+## Example: Run a Server within a VM
+
+Since the context of `.runInThisContext()` actually means the V8 context, you
+will need to apply additional objects to it. Say, you want to run a simple
+web server based on the `http` module, you can achieve this by doing something
+like this:
+
```

Comment:
Looking at this again, I'd generally prefer staying away from "you" language... this can be reworded as:

```
The context of `.runInThisContext()` refers to the v8 context. The code passed 
to this VM context will have it's own isolated scope. To run a simple web server 
using the `http` module, for instance, the code passed to the context must either 
call `require('http')` on it's own, or have a reference to the `http` module passed 
to it. For instance:
```